### ISSUE TITLE
bazel 0.27.1 support

### ISSUE THREAD

**Issue by ptarjan** [bazelbuild/rules_go#2134]
### What version of rules_go are you using?

0.18.7

### What version of gazelle are you using?

0.13.0

### What version of Bazel are you using?

0.27.1

### Does this issue reproduce with the latest releases of all the above?

Yes.

### What operating system and processor architecture are you using?

OS X. X86.

### What did you do?

Tried to upgrade bazel to 0.27.1.

### What did you expect to see?

It passing

### What did you see instead?

```
ERROR: /private/var/tmp/_bazel_pt/6f90a73cb5b664fcd4a48a7e340141d9/external/io_bazel_rules_go/BUILD.bazel:62:1: in go_context_data rule @io_bazel_rules_go//:go_context_data:
Traceback (most recent call last):
	File "/private/var/tmp/_bazel_pt/6f90a73cb5b664fcd4a48a7e340141d9/external/io_bazel_rules_go/BUILD.bazel", line 62
		go_context_data(name = 'go_context_data')
	File "/private/var/tmp/_bazel_pt/6f90a73cb5b664fcd4a48a7e340141d9/external/io_bazel_rules_go/go/private/context.bzl", line 396, in _go_context_data_impl
		cc_common.configure_features(cc_toolchain = cc_toolchain, reque..., ...)
Incompatible flag --incompatible_require_ctx_in_configure_features has been flipped, and the mandatory parameter 'ctx' of cc_common.configure_features is missing. Please add 'ctx' as a named parameter. See https://github.com/bazelbuild/bazel/issues/7793 for det
```

**Comment by ptarjan** [comment]
Sorry for the false alarm. It looks like there was PEBKAC and I didn't actually update you to 0.18.7 in my testing.
